src/migrations/00053-build-stats.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "peak memory usage" INT NULL;
+
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "average memory usage" INT NULL;
+
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "peak cpu usage" INT NULL;
+
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "average cpu usage" INT NULL;
+
+ALTER TABLE "release"
+ADD COLUMN IF NOT EXISTS "run time" INT NULL;
```

Comment:
NIT: I think you can do:
```suggestion
ALTER TABLE "release"
ADD COLUMN IF NOT EXISTS "peak memory usage" INT NULL,
ADD COLUMN IF NOT EXISTS "peak cpu usage" INT NULL,
ADD COLUMN IF NOT EXISTS "average memory usage" INT NULL,
ADD COLUMN IF NOT EXISTS "average cpu usage" INT NULL,
ADD COLUMN IF NOT EXISTS "run time" INT NULL;
```

Reply:
done